src/main/java/edu/cornell/kfs/concur/rest/jsonObjects/ConcurRequestV4ReportDTO.java

```diff
@@ -0,0 +1,304 @@
+package edu.cornell.kfs.concur.rest.jsonObjects;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ConcurRequestV4ReportDTO {
+
+    @JsonProperty("href")
+    private String href;
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("approvalStatus")
+    private ConcurRequestV4StatusDTO approvalStatus;
+
+    @JsonProperty("approved")
+    private boolean approved;
+
+    @JsonProperty("approver")
+    private ConcurRequestV4PersonDTO approver;
+
+    @JsonProperty("businessPurpose")
+    private String businessPurpose;
+
+    @JsonProperty("canceledPostApproval")
+    private boolean canceledPostApproval;
+
+    @JsonProperty("closed")
+    private boolean closed;
+
+    @JsonProperty("creationDate")
+    private Date creationDate;
+
+    @JsonProperty("custom1")
+    private ConcurRequestV4CustomItemDTO chart;
+
+    @JsonProperty("custom2")
+    private ConcurRequestV4CustomItemDTO Account;
+
+    @JsonProperty("endDate")
+    private Date endDate;
+
+    @JsonProperty("everSentBack")
+    private boolean everSentBack;
+
+    @JsonProperty("isUserReviewed")
+    private boolean isUserReviewed;
+
+    @JsonProperty("lastModified")
+    private String lastModified;
+
+    @JsonProperty("mainDestination")
+    private ConcurRequestV4MainDestinationDTO mainDestination;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("owner")
+    private ConcurRequestV4PersonDTO owner;
+
+    @JsonProperty("pendingApproval")
+    private Boolean pendingApproval;
```

Comment:
This property should also be converted from Boolean wrapper to boolean primitive.

Reply:
ok